configs/simple_CNN.json

```diff
@@ -0,0 +1,9 @@
+{
+  "exp_name": "simple_CNN",
+  "num_epochs": 10,
+  "num_iter_per_epoch": 100,
```

Comment:
Number of iterations equal 50 gives similar results, but double the performance. Please change.